moveit_setup_assistant/src/tools/moveit_config_data.cpp

```diff
@@ -78,6 +77,15 @@ MoveItConfigData::~MoveItConfigData()
 {
 }
 
+// ******************************************************************************************
+// Load a robot model
+// ******************************************************************************************
+void MoveItConfigData::setRobotModel(robot_model::RobotModelPtr robot_model)
+{
+  robot_model_ = robot_model;
+  robot_model_const_ = robot_model;
```

Comment:
just curious: do you know why i originally had to add a const and non-const version of robot_model? this is very hacky. shame on me (~6 years ago)

Reply:
Well the `robot_model_const_` is only used in `getRobotModel`, while `robot_model_` is used whenever we are to edit sth in the robot model.

We can remove `robot_model_const_` and just `const_cast` the `robot_model_` in the `getRobotModel()`, does that sound better?